include/crm/common/output.h

```diff
@@ -148,7 +148,7 @@ struct pcmk__output_s {
     /*!
      * \brief The name of this output formatter.
      */
-    char *fmt_name;
+    const char *fmt_name;
```

Comment:
I'm not sure whether I can get away with doing this, or whether it's part of unchangeable public API now.  There's really no reason for fmt_name to not be const.

Reply:
output.h is entirely internal, it's even nodist